interfaces/IBF-dashboard/src/app/components/disaster-type/disaster-type.component.ts

```diff
@@ -11,6 +12,8 @@ import { CountryService } from '../../services/country.service';
 })
 export class DisasterTypeComponent implements OnInit, OnDestroy {
   public disasterTypes: DisasterType[] = [];
+  public disasterTypeMap = DISASTER_TYPES_SVG_MAP
+  public selectedDisasterType: string = null;
```

Comment:
I assumed that there is always a default disaster type based on the selected country. A situation where there is no selected disaster type probably never occurs.

Please use [Disaster Type](https://github.com/rodekruis/IBF-system/blob/master/interfaces/IBF-dashboard/src/app/models/country.model.ts#L43) as the type for `selectedDisasterType`.

Reply:
Taken care